### ISSUE TITLE
Command-line interface for rofication

### ISSUE THREAD

**Issue by kgilmer** [regolith-linux/regolith-desktop#601]
Would resolve: https://github.com/DaveDavenport/Rofication/issues/7

* command should be composable...eg being able to grep notifications and then use the filtered list to say delete those notifications.
* Should provide same workflow as rofi dialog
* Should not introduce large new runtime dependencies

**Comment by ploum** [comment]
If I’m correct, it’s only a matter of writing a python script called "rofication-cli". 

I could make an attempt at it, Could you expand on "should provide same workflow as rofi dialog" ?

**Comment by kgilmer** [comment]
Hi @ploum , I should have updated this issue earlier.  I have [written a cli version](https://github.com/regolith-linux/lago) that will be available in Regolith 2.

It's pretty simple, and was more of a boostrapping exercise for the Notifications view in [Ilia](https://github.com/regolith-linux/ilia).  You're more that welcome to work on it if you like. :smile:

**Comment by ferdinandyb** [comment]
I noticed that `lago` is only mentioned in the release notes. Shouldn't we make a place on the website where we could list and describe in short any tools the regolith ppa provides that is not the DE itself and/or might make sense to use without regolith?

I also only came across `regolith-diagnostic` in the release notes.

I'm also unsure if having this issue is still relevant or can be closed (maybe after adding documentation).

**Comment by kgilmer** [comment]
Yes, both of those items need to be documented on the site @ferdinandyb
